MIGRATION.md

```diff
@@ -2217,3 +2284,62 @@ import {
   <SkeletonText />
 </SkeletonContainer>;
 ```
+
+## Pill
+
+The API of the Pill component has not changed.
+
+### How to migrate your Pill components
+
+To migrate your v3 `Pill` components to v4 run the following [codemod](https://github.com/contentful/forma-36/tree/forma-v4/packages/forma-36-codemod):
+
+`npx @contentful/f36-codemod`
+
+If you want to manually migrate your Pill component to v4, you must update the import. It changes from this:
+
+```tsx
+import { InfoCircleIcon } from '@contentful/f36-icons';
+import { Pill } from '@contentful/forma-36-react-components';
+
+<Pill label="test label" variant="active" onClose={() => {}} />;
+
+<Pill label="test label" variant="deleted" onClose={() => {}} />;
+
+<Pill
+  label="example.user@contentful.com"
+  variant="idle"
+  onDrag={() => {}}
+  dragHandleComponent={
+    <InfoCircleIcon
+      aria-label="Drag handler"
+      variant="muted"
+      style={{ padding: '0.375rem 0.625rem' }}
+    />
+  }
+/>;
+```
+
+to this:
+
+```tsx
+import { InfoCircleIcon } from '@contentful/f36-icons';
+
+import { Pill } from '@contentful/f36-components';
+
+<Pill label="test label" variant="active" onClose={() => {}} />;
+
+<Pill label="test label" variant="deleted" onClose={() => {}} />;
+
+<Pill
+  label="example.user@contentful.com"
+  variant="idle"
+  onDrag={() => {}}
+  dragHandleComponent={
+    <InfoCircleIcon
+      aria-label="Drag handler"
+      variant="muted"
+      style={{ padding: '0.375rem 0.625rem' }}
+    />
+  }
+/>;
+```
```

Comment:
Do we really need to write this part if there is nothing to be done?

We could just tell the user to update the imports and make the guide shorter and easier to read

Reply:
I guess it will be easier to read